packages/core/core-flows/src/order/workflows/return/confirm-receive-return-request.ts

```diff
@@ -232,9 +238,26 @@ export const confirmReturnReceiveWorkflow = createWorkflow(
           }
         })
 
+        const hasReceivedAllItems = retItems.every((item) => {
+          const received = itemUpdates[item.item_id]
+            ? itemUpdates[item.item_id].received_quantity
+            : item.received_quantity
+
+          return MathBN.eq(received, item.quantity)
+        })
+        const updateReturnData = hasReceivedAllItems
+          ? { status: ReturnStatus.RECEIVED, received_at: new Date() }
+          : { status: ReturnStatus.PARTIALLY_RECEIVED }
```

Comment:
comment: This might change if we choose to remove partial receival. Just adding a comment in case we reach a conclusion before the PR is merged so we can get it updated.

Reply:
I'm just bringing what we already have done for the complete return flow
`packages/modules/order/src/services/actions/create-return.ts`

If you guys decide to remove this feature, I can remove even other things like the received_quantity from the return model, etc.
I personally wouldn't remove features that could be used.